src/lru.js

```diff
@@ -98,7 +98,7 @@ class LRU {
 		return Object.keys(this.items);
 	}
 
-	set (key, value, bypass = false, resetTtl = false) {
+	set (key, value, bypass = false, resetTtl = true) {
```

Comment:
this isn't what i meant earlier; it should come from an attribute like 'ttl'.

Reply:
you mean a numeric one?